test/CMakeLists.txt

```diff
@@ -673,7 +673,7 @@ COMMAND ${DYNAMIC_IMPLICITGEMM_BWD_ENVS} $<TARGET_FILE:test_conv2d> --verbose --
 COMMAND ${DYNAMIC_IMPLICITGEMM_BWD_ENVS} $<TARGET_FILE:test_conv2d> --verbose --input 128  256 56 56 --weights 64  256 1 1 --pads_strides_dilations 0 0 1 1 1 1 --disable-forward --disable-backward-weights
 )
 
-add_custom_test(test_conv_igemm_dynamic_xdlops SKIP_UNLESS_ALL
+add_custom_test(test_conv_igemm_dynamic_xdlops_bwd SKIP_UNLESS_ALL
```

Comment:
Does this xDLOPs test excluded from non-gfx908 testing runs? (Ditto others)

Reply:
Hi, Artem,
I do not fully know how to exclude a test from non-gfx908 testing.
Could you please check if they are now excluded from non-gfx908 testing?

Reply:
> I do not fully know how to exclude a test from non-gfx908 testing.

IIRC `cmake ... -DMIOPEN_TEST_GFX908=On ...` tells that we build the library & tests for running on gfx908. Please make sure that gfx908-specific tests DO NOT run when this cmake variable is NOT set.

Reply:
...then please check to see if it really works as expected ;)